{"name": "test_fixture_zcc2[cc_calculator_shuju1]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0.5 == -3\n  +0.5\n  --3", "trace": "self = <cc_work.test_fixture_zcc.TestCal_cc_new object at 0x00000000039ABB00>, cc_calculator_func = <cc_work.calculator_cc.Calculator object at 0x00000000039B4198>, cc_calculator_shuju = [-1, -2, -3]\n\n    @pytest.mark.run(order=4)\n    def test_fixture_zcc2(self, cc_calculator_func, cc_calculator_shuju):\n        result = cc_calculator_func.div(cc_calculator_shuju[0], cc_calculator_shuju[1])\n        print('这是测试用例2')\n        # print(result)\n>       assert result == cc_calculator_shuju[2]\nE       assert 0.5 == -3\nE         +0.5\nE         --3\n\ntest_fixture_zcc.py:32: AssertionError"}, "parameters": [{"name": "cc_calculator_shuju", "value": "[-1, -2, -3]"}], "start": 1609398358097, "stop": 1609398358097, "uuid": "acee5e79-2932-4c49-99fd-e75a6066d238", "historyId": "39396ecd6c851cce595fb0981ae0f9a5", "testCaseId": "6207987451ade26d46e5eac58d4c08e9", "fullName": "test_fixture_zcc.TestCal_cc_new#test_fixture_zcc2", "labels": [{"name": "suite", "value": "test_fixture_zcc"}, {"name": "subSuite", "value": "TestCal_cc_new"}, {"name": "host", "value": "admin-PC"}, {"name": "thread", "value": "12680-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_fixture_zcc"}]}